Code/Mantid/Framework/DataHandling/src/ProcessDasNexusLog.cpp

```diff
@@ -445,7 +447,9 @@ void ProcessDasNexusLog::writeLogtoFile(API::MatrixWorkspace_sptr ws,
   Kernel::Property *log = ws->run().getProperty(logname);
   Kernel::TimeSeriesProperty<double> *tslog =
       dynamic_cast<Kernel::TimeSeriesProperty<double> *>(log);
-  std::vector<Kernel::DateAndTime> times = tslog->timesAsVector();
+  std::vector<Kernel::DateAndTime> times;
```

Comment:
This won't address the problem fully as tslog is still dereferenced a second time, which is unchecked:
```c++
Kernel::TimeSeriesProperty<double> *tslog = dynamic_cast<Kernel::TimeSeriesProperty<double> *>(log);
std::vector<Kernel::DateAndTime> times;
if (tslog)
  tslog->timesAsVector();
std::vector<double> values = tslog->valuesAsVector();
```

Perhaps instead something like this:
```c++
Kernel::TimeSeriesProperty<double> *tslog = dynamic_cast<Kernel::TimeSeriesProperty<double> *>(log);
if (!tslog)
  throw std::runtime_error("Invalid time series log");
std::vector<Kernel::DateAndTime> times = tslog->timesAsVector();
std::vector<double> values = tslog->valuesAsVector();
```
The exception itself is just a placeholder, but if those values aren't being extracted we're probably wanting to fail.